GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg

```diff
@@ -117,19 +127,21 @@ PART
 	//	============================================================================
 	//	Modules and Resources
 	//	============================================================================
-
+	
+	//Same structure as main HS, use lunar HS settings
+	//This actually survives LEO reentry on it's own. Probably not ideal
 	MODULE
 	{
 		name:NEEDS[!DeadlyReentry] = ModuleAblator
 		name:NEEDS[DeadlyReentry] = ModuleHeatShield
 		ablativeResource = Ablator
 		outputResource = CharredAblator
-		outputMult = 0.75
+		outputMult = 0.8
```

Comment:
Why the change in conversion rate?  I think the RO tool uses 0.75.  I don't know the basis for that number, question is more about consistency.

Reply:
The RO tool uses 0.8 (at least, that's what it ends up as in the cache).